components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/AddonsManagerAdapter.kt

```diff
@@ -191,13 +196,27 @@ class AddonsManagerAdapter(
             }
         }
 
-        scope.launch {
-            val iconBitmap = addonCollectionProvider.getAddonIconBitmap(addon)
+        fetchIcon(addon, holder.iconView)
+    }
 
-            iconBitmap?.let {
-                MainScope().launch {
-                    holder.iconView.setImageBitmap(it)
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    @Suppress("TooGenericExceptionCaught")
+    internal fun fetchIcon(addon: Addon, iconView: ImageView, scope: CoroutineScope = this.scope): Job {
+        return scope.launch {
+            try {
+                val iconBitmap = addonCollectionProvider.getAddonIconBitmap(addon)
+                iconBitmap?.let {
+                    scope.launch(Main) {
+                        iconView.setImageBitmap(it)
+                    }
+                }
+            } catch (e: Exception) {
+                scope.launch(Main) {
+                    val context = iconView.context
+                    val att = context.theme.resolveAttribute(android.R.attr.textColorPrimary)
+                    iconView.setColorFilter(ContextCompat.getColor(context, att))
```

Comment:
We need to change the tint of the default icon as if we don't it is not going to adapt to different themes ( like Fenix dark or light [ Video testing on Fenix with different themes](https://drive.google.com/file/d/1kkLrg7CgdjdU-askWub-t1-6ABvGvArw/view?usp=sharing)) 

Causing an issue lbe:
![Screenshot_20200206-184300](https://user-images.githubusercontent.com/773158/73995057-86a80f80-4925-11ea-9df8-6713c7d54e9c.png)